UE/Application/Ports/TimerPort.hpp

```diff
@@ -19,6 +20,9 @@ class TimerPort : public ITimerPort
     void stopTimer() override;
 
 private:
+    void timerThread(Duration duration);
```

Comment:
Could we somehow use `chrono` here? Or just rename the argument name to `durationInMiliseconds`, so we know which unit to use?

Reply:
It is ok with me because we don't really need to know which unit is in use. We must use `using namespace std::chrono_literals` before. You can use different units: https://github.com/Koci0/Nokia-PK-2021/blob/5282beefc1d04bedb2da64657138a4e1b4e2fb3a/UE/Application/States/ConnectedState.cpp#L21-L22